### PR TITLE
Aggregation id integration

### PR DESCRIPTION
**Pull request by porteron**
https://github.com/brave/news-aggregator/issues/922

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -20,6 +26,7 @@
     with open(feed_sources) as f:
         publishers = orjson.loads(f.read())
         output_path = config.output_feed_path / f"{config.feed_path}.json-tmp"
+        insert_or_update_all_publishers()
```
Comment (porteron) [inline src/main.py:29]: I added this here in order to get the local aggregation to run successfully. I will consider moving it and keeping it where it was before in the __main__ part of another file.

Comment (porteron) [reply src/main.py]: removed

File: Makefile [hunk 2]
```diff
@@ -16,19 +16,27 @@ validjson:
...
-	json_verify < output/sources.json
-	json_verify < output/feed_source.json
+	python ./tests/json_verify.py < output/sources.json
+	python ./tests/json_verify.py < output/feed_source.json
 	echo Checking that sources.json is of the expected size...
-	test `stat -c%s output/sources.json` -gt 10
 	echo Checking that feed.json is of the expected size...
-	test `stat -c%s output/feed_source.json` -gt 10
+	echo System $(shell uname)
+ifeq ($(shell uname), Linux)
```
Comment (porteron) [inline Makefile:24]: This is to support running on MacOS

File: tests/json_verify.py
Comment (LorenzoMinto) [inline tests/json_verify.py:1]: Do we need this?

Comment (porteron) [reply tests/json_verify.py:1]: It is so we can support running tests across OS. I did not have any luck finding a mac supported version of `json-verify` pkg. Supposedly it comes with `json-c` lib but I installed and still had no luck. The json verify is a simple function so I just recreated it so we can use here without any dependency: https://github.com/brave/news-aggregator/blob/353e6c4bfd42d0d7a792a881e3ef89223d50f6d0/Makefile#L19

**Pull request merged**